wunderio/nextjs-project
# Update npm packages

## Link to feature environment:

  https://update-npm-packages.nextjs-project.dev.wdr.io

  Basicauth username: silta
  Basicauth password: demo

## Changes proposed in this PR:

Updated following npm packages:
- Production dependencies: `next`, `react` and `react-dom`
- Dev dependencies: `@types/node`, `@types/react`, `@typescript-eslint/eslint-plugin`, `@typescript-eslint/parser`, `eslint`, `eslint-config-next`, `eslint-config-prettier`, `eslint-plugin-prettier`, `eslint-plugin-simple-import-sort`, `prettier`, `typescript`

## How to test:

1. Try installing - `npm i` and make sure npm reports 0 vulnerabilities
2. Try building and starting app - `npm run build && npm run start`
3. Try development build - `npm run dev`
4. Try linting - `npm run lint`
5. Check CircleCI - build - https://app.circleci.com/pipelines/github/wunderio/nextjs-project/607/workflows/c2b75289-2b0e-4d50-9523-159f45fb9200/jobs/794
